tests/python/gaia-ui-tests/gaiatest/tests/unit/test_override_defaults.py

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest import DEFAULT_SETTINGS
+
+TEST_DEFAULTS = {'ftu.manifestURL': 'app://ftu.gaiamobile.org/manifest.webapp'}
+
+
+class TestOverrideDefaults(GaiaTestCase):
+
+    def set_defaults(self):
```

Comment:
One more note, doing it that way also lets the override look at what the parent has already set. That way it can put a conditional on it, or set a "derived default" based on the originals from testvars, or whatever. Using a basic method override means it has to fly blind, or know the parent's implementation, neither of which is great.